packages/portal-proto/src/features/proteinpaint/ProteinPaintWrapper.tsx

```diff
@@ -56,7 +56,7 @@ export const ProteinPaintWrapper: FC<PpProps> = (props: PpProps) => {
       setPickedCases(cases);
       setShowCreateCohort(true);
     },
-    [createSet, pickedCases],
+    [],
```

Comment:
@craigrbarnes  Just wanted to bring to your attention that linter was complaining here for unnecessary deps. Please verify.